source/common.ts

```diff
@@ -370,6 +370,10 @@ const i18n = {
         ),
     );
 
+    if (typeof translation === 'object') {
+      return globalThis.JSON.stringify(translation);
```

Comment:
That's definitely **not** something that we'd like to do here. Let me cite #142:

> Then createReactiveTranslator('a') should return the object { b1: "c1", b2: "c2" } but instead it returns the string '[object Object]'. I think this is a change in behavior from a previous version of the library.

Therefore, we should return the `translation` here. However, I'm not certain that it was actually working like that in the previous versions. @piotrpospiech, could you check it?

Reply:
I checked some previous versions and it didn't work there also.

Reply:
@brianlukoff Can you check that on your end? It'd be a nice feature to have something like that, but it's technically a breaking change. Also, `createReactiveTranslator` is kind of `getTranslation` call (that always returns string) and there's no `getTranslations` counterpart of it.

Reply:
@radekmie I just tested with 1.31.0 and `i18n.createReactiveTranslator(null, 'en-US')([key])` does return an object when *key* corresponds to an object. Our need for this behavior is not to actually retrieve the entire object per se but just to detect that it is an object -- is there another way we can do that?

Reply:
In this case, we will fix that - there was no intention of changing that. @piotrpospiech Can you take care of it?

Reply:
@brianlukoff I tried to reproduce that and I can't.
I use this code:
```js
const T = i18n.createReactiveTranslator(null, 'en-US')('test.a')
console.log('T', T)
```
with this language file:
```yml
_locale: en-US
_namespace: test

welcome: welcome
a:
  b1: b111
  b2: b222
```

I tested this on v1.31.0 and I always get `[object Object]` as result. 

If you can send me a reproduction code, I will continue work on this issue.